definition/remarks_files.md

```diff
@@ -0,0 +1,28 @@
+## What is a remarks file?
+
+To capture special cases, unique administrative hierarchies, and intricacies of various records in Who's On First, we have introduced "remarks" files. Think of remarks files as a unique change log for a given record.
+
+As of writing this, Who's On First maintains just one remarks file, the `1158857531-remarks.md` file for [St. Petersburg, Russia](https://whosonfirst.mapzen.com/spelunker/id/1158857531), though there will inevitably be more remarks files in the future.
```

Comment:
For example, Who's On First maintains a remarks file for [St. Petersburg, Russia](https://whosonfirst.mapzen.com/spelunker/id/1158857531) in `1158857531-remarks.md`.